src/io/common.jl

```diff
@@ -21,6 +21,15 @@ function parse_file(file::String; import_all::Bool=false, vmin::Float64=0.9, vma
 end
 
 
+""
+function parse_file(filename::String; kwargs...)
+    tppm_data = open(filename) do io
+        parse_file(io; kwargs...)
```

Comment:
should this be `parse_file`?  Seems more general, like `parse`.

Reply:
well, I would say that because it only expects file paths, `parse_file` would be the more appropriate name for this one, and the one that takes `IOStream`s would then become `parse` if you wish to follow that kind of convention.

Reply:
Yes, I meant to refer to the function called in line 27, `parse_file(io; kwargs...)` which takes an io as input.

Reply:
I just tried to do this, and it somehow completely breaks the internal `parse(Type, Str)` command which most of the code relies upon to parse things, so I propose we leave it as is for now, and I can see if I can get it to work later.

Reply:
Fine by me.